test/new-e2e/tests/process/windows_test.go

```diff
@@ -63,6 +63,23 @@ func (s *windowsTestSuite) TestProcessCheck() {
 	assertProcessCollected(t, payloads, false, "MsMpEng.exe")
 }
 
+func (s *windowsTestSuite) TestProcessCheck() {
+	s.testProcessCheck()
+}
```

Comment:
Why the wrapped call here?

Reply:
`TestProcessCheck` and `TestProcessChecksInCoreAgent` are nearly the exact same test except for the config be different. So I moved the common aspects to a helper. Maybe a bit overkill, so I'm open to to removing it.

Reply:
Made a more obvious helper with `assertProcessCheck`